### PR TITLE
Hl7 26 po unique uploads

### PR DESCRIPTION
**Pull request by Perry5**
- https://jira.amida.com/browse/HL7-26

- Added logic to prevent uploading file with the same name

- Added a changelog file
- Moved the jwt exp time to as a config var

- Yes, please add 
   - `JWT_EXP_TIME`  and
   - `FILE_UPLOAD_PATH`

- Upload a file (see that it saves to the FS as the user collection
- Edit the file if you like (leaving the filename the same) then re-upload that file. You should get a conflicting error.
- Check that the file doesn't get saved

- N/A

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,5 +1,6 @@
 NODE_ENV=development
 PORT=4040
-JWT_SECRET=0a6b944d-d2fb-46fc-a85e-0295c986cd9f
 MONGO_HOST=mongodb://localhost/hl7-telescoper-api-server-development
 MONGO_PORT=27017
+JWT_SECRET=0a6b944d-d2fb-46fc-a85e-0295c986cd9f
+JWT_EXP_TIME=24h
```
Comment (orndorffgrant) [inline .env.example]: Let's set this to a safe default like `1hr` (I see you did that in the Joi config, but there's a risk that this example `.env` will be copy/pasted and not looked at closely).

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Changelog
```
Comment (orndorffgrant) [inline CHANGELOG.md:1]: Changelog! woo!

File: server/hl7/hl7.controller.js [hunk 3]
```diff
@@ -7,13 +7,22 @@ const Hl7Parser = require('health-level-seven-parser');
 
 const hl7Parser = new Hl7Parser.Hl7Parser();
 
+const uploadedFilePath = 'data/hl7-uploads';
```
Comment (orndorffgrant) [inline server/hl7/hl7.controller.js]: this should probs be an envvar also

**Review by orndorffgrant — CHANGES_REQUESTED** [review]
code review done. I think the default expiry time in `.env.example` is a definite change, the other envvar suggestion I made is optional/arguably out of scope.
Will do functional test presently.

File: .env.example (hunk 1 above)
Comment (Perry5) [reply .env.example]: haha, this wasn't meant to make it here. I did 24h on my .env and forgot to change it here

File: server/hl7/hl7.controller.js (hunk 3 above)
Comment (Perry5) [reply server/hl7/hl7.controller.js]: True- will update

**Comment by Perry5** [comment]
@orndorffgrant would it be possible to add logic to the `tools/dropDatabase.js` so it deletes files that are uploaded when we run test? I am asking here because this change will somewhat break tests

**Comment by orndorffgrant** [comment]
@Perry5 good idea, should I commit that to this branch?

**Comment by Perry5** [comment]
@orndorffgrant lets make a separate branch for it.

**Review by orndorffgrant — APPROVED** [review]

**Pull request merged**